pkg/dockerregistry/server/repositorymiddleware.go

```diff
@@ -147,10 +155,21 @@ func newRepositoryWithClient(
 		namespace:        nameParts[0],
 		name:             nameParts[1],
 		pullthrough:      pullthrough,
+		acceptschema2:    acceptschema2,
 		cachedLayers:     cachedLayers,
 	}, nil
 }
 
+func getBoolOption(name string, defval bool, options map[string]interface{}) bool {
+	if value, ok := options[name]; ok {
+		var b bool
```

Comment:
@miminar say Hello to `go tool vet -shadow -shadowstrict ./repositorymiddleware.go` in `hack/verify-govet.sh`